doc: enable `--generate-cmd` and update docs (#2334)
aws/copilot-cli: site/content/docs/commands/task-run.en.md

  --env-vars stringToString        Optional. Environment variables specified by key=value separated by commas. (default [])
  --execution-role string          Optional. The role that grants the container agent permission to make AWS API calls.
  --follow                         Optional. Specifies if the logs should be streamed.
  --generate-cmd string            Optional. Generate a command with a pre-filled value for each flag.
                                   To use it for an ECS service, specify --generate-cmd <cluster name>/<service name>.

>                                    Alternatively, if the service is created with Copilot, specify --generate-cmd <application>/<environment>/<service>.
>                                    Similarly, to use it for a job created with Copilot, specify --generate-cmd <application>/<environment>/<job>.
>                                    Cannot be specified with any other flags.

Reviewer: nit: 
```suggestion
                                   Alternatively, if the service or job is created with Copilot, specify --generate-cmd <application>/<environment>/<name>.
                                   Cannot be specified with any other flags.
```
